controller/idgen/id.go

```diff
@@ -56,9 +56,10 @@ func (self *shortIdGenerator) NextAlphaNumericPrefixedId() (string, error) {
 	}
 }
 
-func NewUUIDString() (string, error) {
+func NewUUIDString() string {
 	id := uuid.New()
 	v := &big.Int{}
 	v.SetBytes(id[:])
-	return basex.EncodeInt(v)
+	result, _ := basex.EncodeInt(v)
+	return result
 }
```

Comment:
Doesn't `uuid` support `uuid.NewString()` or does this do something different.

Reply:
It uses a denser encoding (fewer characters) than the default uuid encoding